modules/nodes-cma-autoscaling-custom-creating-workload.adoc

```diff
@@ -125,16 +125,11 @@ spec:
       threshold: '5'
       query: sum(rate(http_requests_total{job="test-app"}[1m]))
       authModes: basic
-  - authenticationRef: <17>
+    authenticationRef: <17>
       name: prom-triggerauthentication
-    metadata:
-      name: prom-triggerauthentication
-    type: object
-  - authenticationRef: <18>
-      name: prom-cluster-triggerauthentication
-    metadata:
+    authenticationRef: <18>
```

Comment:
We still have 2 `authenticationRef` items in this section. If you think they both should appear here as a reference, then make sure to re-write the note below "It is not necessary to specify both a namespace trigger authentication and a cluster trigger authentication." to say that you should only specify one or the other, not both.

Reply:
@joelsmith Thanks! I think we got it now.

Reply:
I must not be seeing the updates. I still see two `authenticationRef` fields (17 & 18) and the note has not been updated. I see the same thing in `modules/nodes-cma-autoscaling-custom-creating-job.adoc` (14 & 15).

Instead of having two `authenticationRef` fields, could we just have a single one, but have a callout on the `kind` sub-field which tells them that it can be `ClusterTriggerAuthentication` or `TriggerAuthentication` and that it defaults to `TriggerAuthentication`?

Reply:
@joelsmith I fear the preview was slow to update. Sorry to have wasted your time. I think we got it now. I took your suggestion here.